### DISCUSSION TITLE
Feature Requests & Ideas

### DISCUSSION THREAD

**Discussion by oricalworks** [Circuit-Pro/openpilot#6] [Ideas]
Post your Ideas and code here (Cherry Picked commits are the best)

**Comment by Virginian** [comment]
It would be nice on Vision Long to allow coasting 5mph over the set speed or closing the distance to the car in front of you before braking.

**Comment by moreno7798** [comment]
On the C2 "Device Options", move "Reboot" and "Power Off" buttons to the very top. This way it's easier to reboot device without having to scroll all the way down first.
